### PR TITLE
Issue comments

### PR DESCRIPTION
**Pull request by KPostOffice**
create comments in issue

## This introduces a breaking change

- [x] Yes
- [ ] No

### CODE REVIEW

File: git_app/cli.py [hunk 1]
```diff
@@ -91,18 +91,18 @@ def cli(
...
 ):
     """Run base command for varangian git forge application."""
-    lib.run(
+    lib.create_issue_with_comments(
         repo=repo,
         predictions_file=predictions_file,
         trace_directory=trace_directory,
         namespace=namespace,
-        max_count=max_count,
+        # max_count=max_count,
```
Comment (harshad16) [inline git_app/cli.py]: Is commenting a line in function arguments allowed?
will this not cause any issu?

**Review by harshad16 — COMMENTED** [review]
lgtm
just one question

File: git_app/cli.py (hunk 1 above)
Comment (KPostOffice) [reply git_app/cli.py]: I will remove this line

Comment (harshad16) [reply git_app/cli.py]: we might have to change it in other places in the pr , the other function are also defined with comment inside the argument, please bring them outside of the function definition.

**Pull request closed without merging**